blog/2019-10-08-handling-unchanged-postgres-toast-values.adoc

```diff
@@ -0,0 +1,316 @@
+= Strategies for Handling Unchanged Postgres TOAST Values
+gmorling
+:awestruct-tags: [ discussion, examples, postgres, kafka-streams ]
+:awestruct-layout: blog-post
+
+Let's talk about TOAST.
+Toast?
+No, TOAST!
+
+So what's that?
+https://www.postgresql.org/docs/current/storage-toast.html[TOAST] (The Oversized-Attribute Storage Technique) is a mechanism in Postgres which allows to store large column values in multiple physical rows, circumventing the page size limit of 8 KB.
+
+++++
+<div class="imageblock centered-image">
+    <img src="/images/postgres_toast.png" style="max-width:100%; width:40%; margin-bottom:10px; margin-top:10px;" class="responsive-image" alt="TOAST!">
```

Comment:
A joy of different associations :-) https://images2.minutemediacdn.com/image/upload/c_fill,g_auto,h_1248,w_2220/f_auto,q_auto,w_1100/v1555357947/shape/mentalfloss/istock_000034973846_small_0.jpg

Reply:
LOL

Reply:
I speant longer than I wanted on finding an image and adding the caption :)